### DISCUSSION TITLE
Welcome to chatify Discussions!

### DISCUSSION THREAD

**Discussion by jeremymanning** [ContextLab/chatify#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HemuManju** [comment]
Hello Everyone, 

I am able to put together a basic python package implementing magic function in IPython notebook which can access ChatGPT API. I can discuss the implementation in detail in the coming Thursday meeting. Looking forward for the discussion. Here is a screenshot of the implementation. 
![test](https://user-images.githubusercontent.com/16517399/234168374-3b4e3f8e-9483-461e-b18d-2d7c1f8e48f3.jpg)

**Reply by jeremymanning** [reply]
That looks awesome @HemuManju!!  Could you please submit a pull request with your demo code?  Make sure not to upload your API key, since the repo is public.

**Reply by HemuManju** [reply]
I have created a pull request
